### PR TITLE
Additional Style Updates

### PR DESCRIPTION
**Pull request by zengshengliu**
Update App "About" page background, commands, and font color
Update App Tabs color
Update Thread "footer" color
Update Button Attachment color
Update Dialog textarea font color
Update `c-pillow_file` font color
Update Threads banner color
Update Threads Flexpane reply box color
Update Threads page color (I hate they have `p-threads` and `p_threads` class 😞)
Update Threads app label and timestamp color
Update Channel Preview footer color
Update Channel Notification Banner color
Update Flexpane Close button color
Update Multi File Upload dialog color
Update File Upload overlay color

### CODE REVIEW

File: scss/modules/modals/_channel.scss [hunk 1]
```diff
@@ -107,3 +107,19 @@
 .notification_prefs_icon::before {
   color: $base-font-color;
 }
+
+.channel_header {
+  .channel_header__tabs {
+    .channel_header__tab {
+      color: $base-link-color;
+
+      &.--active {
```
Comment (mastermel) [inline scss/modules/modals/_channel.scss]: I believe you may have meant for this to be `&--active`? With the period in there after the ampersand it attempts to select on the class `--active` which I don't think Slack uses. Also I'm not sure it's legal css to start a class name with two hyphens.

**Review by mastermel — CHANGES_REQUESTED** [review]

File: scss/modules/modals/_channel.scss (hunk 1 above)
Comment (zengshengliu) [reply scss/modules/modals/_channel.scss]: image
This is the "Apps" section header which does have the `--active` as the class name

Comment (mastermel) [reply scss/modules/modals/_channel.scss]: Well TIL :)

Comment (zengshengliu) [reply scss/modules/modals/_channel.scss]: 😄

**Comment by zengshengliu** [comment]
@mastermel 	Let me know if you have any more questions.  I think I am good on this PR (unless I find more missing styling)

**Review by mastermel — APPROVED** [review]

**Comment by mastermel** [comment]
@zengshengliu I wonder if you could help me out? I'm attempting to test your PR locally using Stylish and the Slack web client in Chrome. When I attempt to import the transpiled css I get [this error](https://www.dropbox.com/s/dkqbst21yy1hcdp/Screen%20Shot%202019-03-08%20at%208.53.41%20AM.png?dl=0). Any ideas?

**Comment by zengshengliu** [comment]
@mastermel 	It looks like Stylish doesn't like the class name `--active`.  Let me see if there is another selector I can use instead, because it does looks like an invalid class name but somehow Slack uses it

**Comment by zengshengliu** [comment]
@mastermel 	That the exact selector Slack used by default.  I don't think there is another selector we can use.  Do you have any suggestion?

image

**Comment by mastermel** [comment]
@zengshengliu sorry for the late reply! I haven't been able to find another way to override that `.channel_header__tab.--active` selector such that it will pass the Mozilla Format 😞 

Without being able to do that I'm not sure we'll be able to publish your changes to userstyles.org.

Do you think for the time being you could comment that override out, so that we can get the rest of your updates merged?

**Comment by zengshengliu** [comment]
@mastermel 	will do that

**Comment by zengshengliu** [comment]
@mastermel 	I will do that shortly

**Comment by zengshengliu** [comment]
@mastermel 	This PR should be good to merge now.

**Comment by nysos3** [comment]
@zengshengliu @mastermel 

I'll be switching to running this fork for some time for some pre-merge testing.

**Review by mastermel — APPROVED** [review]
Changes look good and runs for me locally. I appreciate you cleaning up some of the inline transparent colors with a scss variable!

**Pull request merged**